pkg/instrumentation/get-target-id_test.go

```diff
@@ -155,6 +157,18 @@ func Test_GetTargetId(t *testing.T) {
 
 		assert.Equal(t, actualAbsolute, actualRelative)
 	})
+
+	t.Run("configured repo url", func(t *testing.T) {
```

Comment:
**question:** Do we want to add a check here for behaviour with an empty or malformed git url?

Reply:
Done!